scripts/security-audit.js

```
@@ -0,0 +1,50 @@
+#!/usr/bin/env node
+
+/**
+ * Security audit script for build-time sanitization
+ *
+ * This script runs the sanitization security tests to ensure that
+ * the blog data generation process is properly sanitizing content.
+ */
+
+const { execSync } = require('child_process')
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)